lib/Dialect/TritonGPU/Transforms/WarpSpecialization/PartitionScheduling.cpp

```diff
@@ -151,6 +152,43 @@ static Partition *scheduleUsers(scf::ForOp loop, WarpSchedule &schedule,
   return partition;
 }
 
+// Schedule post-loop operations (operations outside and after the loop) into
+// the epilogue partition. This recursively schedules operations that consume
```

Comment:
This might be a naive question, but are we assuming there is always a singular epilogue partition?